### PR TITLE
Collection configuration MUST be collection.xconf

### PR DESCRIPTION
**Pull request by adamretter**
Closes https://github.com/eXist-db/exist/issues/1282

### CODE REVIEW

File: data/indexing.xml [hunk 1]
```diff
@@ -121,19 +121,13 @@
                 the <filename>/db/foo</filename> hierarchy inside <filename>/db/system/config/</filename>. Subcollections which do not have a <filename>collection.xconf</filename> file of their own will be governed by the
                 configuration policy specified for the closest ancestor collection which does have
                 such a file, so you are not required to specify a configuration for every
-                collection. Note, however, that configuration settings do not cascade. If you choose
-                to deploy a <filename>collection.xconf</filename> file in a subcollection, you must
+                collection. Note, however, that configuration settings do not cascade, that is to say that a sub-Collection
+		    with a <filename>collectio.xconf</filename>, will not inherit any settings from any ancestor Collection
```
Comment (dizzzz) [inline data/indexing.xml]: Typo in .xco file name

Comment (dizzzz) [inline data/indexing.xml]: Collectio iso collection I mean

**Comment by adamretter** [comment]
@dizzzz All fixed :-)

**Comment by adamretter** [comment]
Can we do that in a subsequent PR? It probably needs to be done across all
docs!

**Comment by ljo** [comment]
Sure, I, can do that across all docs.

**Comment by dizzzz** [comment]
oops, by pulling in @ljo this needs rework ....

**Comment by dizzzz** [comment]
reworked @adamretter 's change ....

**Pull request merged**